index.html

```diff
@@ -216,6 +216,90 @@ <h4>Skills</h4>
           </ul>
         </article>
       </section>
+      <section class="form-section">
+        <header class="form-header">
+          <h2
+            >I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.</h2
+          >
+        </header>
+        <form action="https://formspree.io/f/xjvlozll" method="post">
+          <ul class="form-list form-container">
+            <li class="form-list-items">
+              <label for="name">
+                <input
+                type="text"
+                name="Name"
+                id="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+                class="input"
+              />
+              </label>
+            </li>
+            <li class="form-list-items">
+              <label for="email">
+                <input
+                type="email"
+                name="Email"
+                id="email"
+                placeholder="Email address"
+                required
+                class="input"
+              />
+              </label>
+            </li>
+            <li class="form-list-items">
+              <label for="message">
+                <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="11"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+              </label>
+            </li>
```

Comment:
- [ ] The width of the `textarea` is not 100%, I recommentd to use width 100% in style .
-
![Screenshot from 2022-03-22 23-40-36](https://user-images.githubusercontent.com/26767383/159552848-912b00af-ff20-4342-8571-e40ae634120d.png)

Reply:
the style width was 100%, I changed the id name which was what affected my whole item, making the styles not be effective on it, thanks for the review! it is all checked now.

Reply:
All good now 👍🏽